<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="30190" started-at="2019-05-30T06:47:20Z" finished-at="2019-05-30T06:47:50Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="30190" started-at="2019-05-30T06:47:20Z" finished-at="2019-05-30T06:47:50Z">
      <class name="com.scrapPlant.functions.execution">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.scrapPlant.functions.execution@880ec60]" name="beforeTest" is-config="true" duration-ms="17" started-at="2019-05-30T12:17:20Z" finished-at="2019-05-30T12:17:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="Execution()[pri:0, instance:com.scrapPlant.functions.execution@880ec60]" name="Execution" duration-ms="30160" started-at="2019-05-30T12:17:20Z" finished-at="2019-05-30T12:17:50Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.scrapPlant.utilities.ReusableFunctions.renamefile(ReusableFunctions.java:470)
	at com.scrapPlant.utilities.ReusableFunctions.launchBrowser(ReusableFunctions.java:237)
	at com.scrapPlant.functions.execution.Execution(execution.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Execution -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.scrapPlant.functions.execution@880ec60]" name="afterTest" is-config="true" duration-ms="0" started-at="2019-05-30T12:17:50Z" finished-at="2019-05-30T12:17:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.scrapPlant.functions.execution -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
